boards/st/stm32u083c_dk/stm32u083c_dk.dts

```diff
@@ -36,6 +36,20 @@
 	aliases {
 		led0 = &green_led_1;
 	};
+
+	touch_buttons {
+		compatible = "tsc-keys";
+		status = "okay";
```

Comment:
drop, all nodes are enabled by default, this is not extending a disabled node

Reply:
got it, thanks